### DISCUSSION TITLE
[Q&A] ESXi inventory with glpi inventory plugin

### DISCUSSION THREAD

**Discussion by waaalex** [glpi-project/glpi-agent#813] [Q&A] [answered]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

Hello, 

Glpi 10.0.17
Glpi inventory 1.4.0
Glpi agent 1.11

ESX inventory and deploy tasks no longer working. Not supported.
Have you got a clue to help me?

```

<!DOCTYPE html>
<html lang="fr"       >
<head>
   <title>Authentification - GLPI</title>

   <meta charset="utf-8" />
...
[Thu Dec 19 10:55:17 2024][info] Deploy task not supported by server0
[Thu Dec 19 10:55:23 2024][info] running task ESX
[Thu Dec 19 10:55:23 2024][error] [http client] Can't decode JSON content, starting with: 
         

<!DOCTYPE html>
<html lang="fr"       >
<head>
   <title>Authentification - GLPI</title>

   <meta charset="utf-8" />
...
[Thu Dec 19 10:55:23 2024][info] ESX task not supported by server0
[Thu Dec 19 10:55:26 2024][info] running task Inventory
[Thu Dec 19 10:55:26 2024][info] New inventory from SRV-SECU03.get.com-2023-08-13-17-00-44 for server0
[Thu Dec 19 10:55:57 2024][info] target server0: next run: Fri Dec 20 10:33:20 2024 - https://glpi/
[Thu Dec 19 10:55:57 2024][info] GLPI Agent memory usage: WSS=3518464 PFU=173236224
```

Regards.
Alex.

**Comment by g-bougard** [comment, marked as answer]
You seem to reach the GLPI Authentication page. This may mean you glpi-agent `server` configuration is wrong or something was change in the glpi httpd server configuration, maybe a failing redirection.

**Reply by waaalex** [reply]
glpi-agent server configuration is wrong >> YES, https://glpi/plugins/glpiinventory.

**Comment by waaalex** [comment]
HEllo,
Thanks for your answer.
agent settings is ok because agent can reach server to make his own inventory.

Here is my redirection on apache conf file. I ve taken this config from documentation (maybe i missed something?) : 
```
 DocumentRoot /var/www/html/glpi/public
 <Directory "/var/www/html/glpi/public">
     Require all granted
     RewriteEngine On
     RewriteCond %{REQUEST_FILENAME} !-f
     RewriteRule ^(.*)$ index.php [QSA,L]
     Options FollowSymlinks
     AllowOverride All
 </Directory>
```
Regards.

**Comment by waaalex** [comment]
Ok, i apologize, you were right about server setting on glpi agent 👍 .
On server setting, i ve put https://glpi/plugins/glpiinventory.
Shall i change this setting for all glpi agents?
Thanks.

**Reply by g-bougard** [reply]
Hi @waaalex 

there are some important caution explained here: https://glpi-agent.readthedocs.io/en/latest/configuration.html#server

Have you checked them ? the `/plugins/glpiinventory/` part can be used only is you installed glpi-inventory-plugin manually into `glpi/plugins/`. If you didn't installed it, just remove this part and only inventory task can be supported. If you installed it to handle other tasks via marketplace, replace `/plugins/` by `/marketplace/`.

**Reply by waaalex** [reply]
Hello @g-bougard , Yes i ve checked. I installed manually. Thank you very mluch for your help.
